### DISCUSSION TITLE
About the script "Test_F450_Launch.xml" throw failed to tie property

### DISCUSSION THREAD

**Discussion by sethome2** [JSBSim-Team/jsbsim#1183] [General]
Hi there!

I am currently working on simulating the F450 aircraft, which is a type of multi-copter drone. I’ve written the following Python code to attempt this simulation:
```python
import jsbsim

jsbsim.FGJSBBase().debug_lvl = 0
fdm = jsbsim.FGFDMExec('./jsbsim')

fdm.load_script('scripts/Test_F450_Launch.xml')
# or 
# fdm.load_model('F450')

while fdm.run():
    pass
```
The program appears to run normally, but I’m encountering an error message that looks like this:
![output_img](https://github.com/user-attachments/assets/2ab88374-001d-4701-95ed-365a17f0b637)

For the first error, “INCOMPATIBLE CFG FILE,” I suspect it might be due to the F450 CFG label being <fdm_config release="Alpha" version="3.0">, indicating the release is in ALPHA rather than PRODUCTION.
﻿
However, I’m quite puzzled by the second error and would greatly appreciate any insights or suggestions you might have. Thank you in advance for your help!

**Comment by andgi** [comment]
I think it is the version attribute that is wrong: It should be
version="2.0" since we don't have a version 3 yet.

Cheers,
Anders

**Reply by sethome2** [reply]
Thanks a lot!

In repo, the version attribute in F450 is 3.0, it may be a bug.

Also, the error below still exists after changed version attribute to 2.0. 
![Running Screen Shot](https://github.com/user-attachments/assets/8c4247ff-ed42-4083-ae9e-c53f3b8a3807)

I am trying to figure out the reason. If any points I found, I will post here.
